### PR TITLE
update Anaplan Connector

### PR DESCRIPTION
**Pull request by shuang-anaplan**
Update for Anaplan Connector to support Anaplan OAuth APIs for authenticating with Anaplan, submitted by spencer.huang@anaplan.com

cc: @Chanaveer

### CODE REVIEW

File: powerquery-docs/Connectors/Anaplan.md [hunk 1]
```diff
@@ -1,120 +1,201 @@
----
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Anaplan.md:1]: These changes should not be reverted.

File: powerquery-docs/Connectors/Anaplan.md [hunk 2]
```diff
@@ -1,120 +1,201 @@
...
 
-ms.topic: conceptual
-ms.date: 2/28/2022
-ms.author: dougklo
-ms.reviewer: kvivek
-LocalizationGroup: reference
----
+The Anaplan Connector for Power BI Desktop enables access to your Anaplan model exports. It also:
 
-# Anaplan
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Anaplan.md:13]: This title change should not be revereted.

File: powerquery-docs/Connectors/Anaplan.md [hunk 3]
```diff
@@ -1,120 +1,201 @@
...
+-   Runs via Anaplan public data integration APIs.
+-   Loads export actions saved in your Anaplan models, directly into Power BI.
+-   Displays Anaplan models (those not archived), within your default Tenant.
+-   Create datasets in the Power BI Desktop and export or refresh them to the Power BI Service (cloud).
 
->[!Note]
->The following connector article is provided by Anaplan, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the Anaplan website and use the support channels there.
+You configure and authenticate within the Anaplan connector to run and load your export. There are also [system requirements](https://www.microsoft.com/en-us/download/details.aspx?id=58494 "System Requirements") to verify before you install the Microsoft Power BI Desktop. 
 
-## Summary
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Anaplan.md:18]: This summary should not be removed

File: powerquery-docs/Connectors/Anaplan.md (hunk 2 above)
Comment (shuang-anaplan) [reply powerquery-docs/Connectors/Anaplan.md:13]: agree

File: powerquery-docs/Connectors/Anaplan.md (hunk 3 above)
Comment (shuang-anaplan) [reply powerquery-docs/Connectors/Anaplan.md:18]: agree

File: powerquery-docs/Connectors/Anaplan.md (hunk 1 above)
Comment (shuang-anaplan) [reply powerquery-docs/Connectors/Anaplan.md:1]: agree

Comment (shuang-anaplan) [reply powerquery-docs/Connectors/Anaplan.md:1]: agree

File: powerquery-docs/Connectors/Anaplan.md [hunk 4]
```diff
@@ -1,120 +1,294 @@
...
+| Release State                    | General Availability |
+| Products                         | Power BI (Datasets)  |
+| Authentication Types Supported   | Basic, OAuth2        |
+| Function Reference Documentation | -                    |
+|                                  |                      |
 
-## Capabilities supported
+# Anaplan
+
+# Anaplan Connector Power BI Desktop
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Anaplan.md:30]: Remove this

File: powerquery-docs/Connectors/Anaplan.md [hunk 5]
```diff
@@ -1,120 +1,294 @@
...
+-   Displays Anaplan models (those not archived), within your default Tenant.
+-   Create datasets in the Power BI Desktop and export or refresh them to the Power BI Service (cloud).
+
+There are [system requirements](https://www.microsoft.com/en-us/download/details.aspx?id=58494 "System Requirements") to verify before you install the Microsoft Power BI Desktop.
+
+# Get data
+
+To access your saved export from the Power BI desktop, you **Get data**.
+
+(1)) Select **Get data** from the ribbon menu.\
```
Comment (bezhan-msft) [inline powerquery-docs/Connectors/Anaplan.md:45]: Is this proper Markdown bulleting?

**Review by bezhan-msft — CHANGES_REQUESTED** [review]
Updated review

**Comment by bezhan-msft** [comment]
@DougKlopfenstein - could you please take a look at this for an initial review?

**Comment by JasonWHowell** [comment]
The PR is in conflict now. I can't assume which paragraphs should win.

**Comment by DougKlopfenstein** [comment]
Closed due to conflicts - this pull request was copied to the private repo and redone:
[First draft of updated Anaplan connector article #647](https://github.com/MicrosoftDocs/powerquery-docs-pr/pull/647).
The changes were approved and are now live - [Anaplan](https://docs.microsoft.com/power-query/connectors/anaplan)

**Pull request closed without merging**